corrscope/gui/view_mainwindow.py

```diff
@@ -160,13 +160,26 @@ def add_appear_tab(self, s: LayoutStack) -> QWidget:
                 with add_row(s, "", BoundColorWidget) as self.render__bg_color:
                     pass
 
+                with add_row(s, tr("Background image"), QHBoxLayout):
+                    with append_widget(s, BoundLineEdit) as self.render__bg_image:
+                        pass
+                    with append_widget(s, QPushButton) as self.bg_image_browse:
+                        pass
+
                 with add_row(s, "", BoundColorWidget) as self.render__init_line_color:
                     pass
 
                 with add_row(s, "", BoundDoubleSpinBox) as self.render__line_width:
                     self.render__line_width.setMinimum(0.5)
                     self.render__line_width.setSingleStep(0.5)
 
+                with add_row(s, "", BoundColorWidget) as self.render__init_line_outline_color:
+                    pass
+
+                with add_row(s, "", BoundDoubleSpinBox) as self.render__line_outline_width:
+                    self.render__line_outline_width.setMinimum(0.0)
```

Comment:
`BoundDoubleSpinBox` minimum defaults to 0, I think this line may be unnecessary.